spec/system/admin/item_price_policies_controller_spec.rb

```diff
@@ -69,6 +69,6 @@
   end
 
   def table_row(*columns)
-    columns.join("\t")
+    columns.join(" ")
```

Comment:
It seems phantomjs and chrome treat the content to text of tables slightly different.